src/aleph/services/cost.py

```diff
@@ -344,28 +344,29 @@ def _get_execution_volumes_costs(
                 isinstance(content, CostEstimationProgramContent)
                 and content.data.estimated_size_mib
             ):
-                volumes.append(
-                    SizedVolume(
-                        CostType.EXECUTION_PROGRAM_VOLUME_DATA,
-                        Decimal(content.data.estimated_size_mib),
-                        content.data.ref,
+                if content.data.estimated_size_mib:
```

Comment:
this `if` is not necessary and can cause bad cost calculation for `ProgramContent` instances